src/NLog/Internal/FilePathLayout.cs

```diff
@@ -77,6 +77,9 @@ internal class FilePathLayout : IRenderable
 
         private bool _cleanupInvalidChars;
 
+        private string _cachedPrevRawFileName;
+        private string _cachedPrevCleanFileName;
```

Comment:
We should document (xmldoc) that `_cachedPrevCleanFileName` is a pair with `_cachedPrevRawFileName`, right?